### PR TITLE
Footer style update

### LINKED ISSUE #818: Footer Style Update
Relate issue is here https://github.com/comicrelief/comicrelief/issues/643

### PR DESCRIPTION
**Pull request by LouiaR**
Fixes #818

### CODE REVIEW

File: sass/themes/cr/2019/components/navigation/_expanded-footer.scss [hunk 1]
```diff
@@ -27,40 +27,39 @@ div.footer--minimal.footer-nav-outer-wrapper {
 
 div.footer--expanded.footer-nav-outer-wrapper {
   line-height: initial;
-  background: #f2f2f2;
-  padding: 45px 18px 40px;
+  background: $colour-black;
```
Comment (AndyEPhipps) [inline sass/themes/cr/2019/components/navigation/_expanded-footer.scss]: Based on the design, this needs to be #FFF,  this variable is from another theme

File: sass/themes/cr/2019/components/navigation/expanded-footer.twig [hunk 2]
```diff
@@ -63,11 +63,8 @@
               <a href="/" title="A Second Menu" aria-expanded="false" aria-haspopup="true" data-drupal-link-system-path="<front>" class="is-active"><span class="menu-item__text">Hear From Us</span></a>
               <ul class="menu menu--level-0 menu menu--expanded-footer menu--has-2-items menu--level-1 clearfix">
                 <li class="menu-item menu-item--1 clearfix">
-                  <a class="js-esu-popup" href="#">
+                  <a href="#" data-drupal-link-system-path="<front>" class="is-active">
```
Comment (AndyEPhipps) [inline sass/themes/cr/2019/components/navigation/expanded-footer.twig]: We can get rid of the Drupal stuff from here, and probably the active class too

**Comment by AndyEPhipps** [comment]
screen shot 2019-01-16 at 16 09 13
screen shot 2019-01-16 at 16 09 10
Copyright next needs left alignment, plus we need to swap the order of the logo and copyright copy here- I know I said not to update the template, but this is ok :)

--
screen shot 2019-01-16 at 16 05 30
screen shot 2019-01-16 at 16 05 25
Lineheight here looks a little off

**Comment by AndyEPhipps** [comment]
screen shot 2019-01-16 at 16 14 44
screen shot 2019-01-16 at 16 14 41

Social icons need resizing and aligning

File: sass/themes/cr/2019/components/navigation/_expanded-footer.scss [hunk 3]
```diff
@@ -27,40 +27,39 @@ div.footer--minimal.footer-nav-outer-wrapper {
...
   }
-  span.lg-breakpoint {
-    display: none;
-    visibility: hidden;
-  }
-  @include breakpoint($screen-lg) {
-    padding: 65px 18px 60px;
+
+  @include breakpoint($screen-lg) { 
     span.lg-breakpoint {
```
[the remark this answers is not in the document]
Comment (LouiaR) [reply sass/themes/cr/2019/components/navigation/_expanded-footer.scss]: Agree, I the drop menu was not opening on the small screen so decide to keep this

[the remark this answers is not in the document]
Comment (AndyEPhipps) [reply sass/themes/cr/2019/components/navigation/_expanded-footer.scss]: Hm, not sure what you mean, but we can look at this properly tomorrow, and get James' input

File: sass/themes/cr/2019/components/navigation/_expanded-footer.scss (hunk 1 above)
Comment (AndyEPhipps) [reply sass/themes/cr/2019/components/navigation/_expanded-footer.scss]: Haha whoops, yes.. wrong way

File: sass/themes/cr/2019/components/navigation/expanded-footer.twig (hunk 2 above)
Comment (LouiaR) [reply sass/themes/cr/2019/components/navigation/expanded-footer.twig]: Sure

**Pull request merged**